content/ja/docs/concepts/overview/components.md

```diff
@@ -58,11 +58,11 @@ Kubernetesをオンプレミスあるいは個人のPC内での学習環境で
 
 kube-controller-managerと同様に、cloud-controller-managerは複数の論理的に独立したコントロールループをシングルバイナリにまとめ、一つのプロセスとして動作します。パフォーマンスを向上させるあるいは障害に耐えるために水平方向にスケールする(一つ以上のコピーを動かす)ことができます。
 
-次のコントローラーは、クラウドプロバイダーへの依存関係を持つことがあります。
+次の各コントローラは、それぞれ以下に示すような目的のためにクラウドプロバイダへの依存関係を持つことができるようになっています。
```

Comment:
```suggestion
次の各コントローラーは、それぞれ以下に示すような目的のためにクラウドプロバイダーへの依存関係を持つことができるようになっています。
```

Reply:
Sorry, I follow the conventions of the translation about 'er'.